### DISCUSSION TITLE
How to get the retry count on refetch?

### DISCUSSION THREAD

**Discussion by Jurek-Raben** [TanStack/query#7192] [Q&A]
Hi,

I would like to know the refetch try amount. Is there such variable in the query.state?

Currently I am working around it like:

```
refetchInterval: (query) => {

if (query.state['retryCount'] > REFRESH_RETRIES) {
  do something..
}

if (!query.state['retryCount']) {
  query.state['retryCount'] = 0;
}
query.state['retryCount']++;

return REFRESH_INTERVAL;
}
```

But is there a better way?

**Comment by neemski** [comment]
`query.state.dataUpdateCount` is probably what you're looking for
